core/shared/src/main/scala/cats/effect/IO.scala

```diff
@@ -387,8 +387,142 @@ sealed abstract class IO[+A] extends internals.IOBinaryCompat[A] {
   final def to[F[_]](implicit F: LiftIO[F]): F[A @uncheckedVariance] =
     F.liftIO(this)
 
+  /**
+   * Returns an `IO` action that treats the source task as the
+   * acquisition of a resource, which is then exploited by the `use`
+   * function and then `released`.
+   *
+   * The `bracket` operation is the equivalent of the
+   * `try {} catch {} finally {}` statements from mainstream languages.
+   *
+   * The `bracket` operation installs the necessary exception handler
+   * to release the resource in the event of an exception being raised
+   * during the computation, or in case of cancelation.
+   *
+   * If an exception is raised, then `bracket` will re-raise the
+   * exception ''after'' performing the `release`. If the resulting
+   * task gets cancelled, then `bracket` will still perform the
+   * `release`, but the yielded task will be non-terminating
+   * (equivalent with [[IO.never]]).
+   *
+   * Example:
+   *
+   * {{{
+   *   import java.io._
+   *
+   *   def readFile(file: File): Task[String] = {
```

Comment:
Should we use `IO` here? I can merge this and change it later if you want :)

Reply:
Yes, I copy/pasted the ScalaDoc from Monix's Task. Will make changes.

Reply:
OK, done.